docs/src/doc/user-guide/exporting.md

```diff
@@ -103,17 +103,13 @@ On desktop platform default export is inferred by the `godot_kotin_configuration
 ## iOS
 
 !!! warning
-    IOS export is experimental.
+With this export you don't have a choice regarding JVM version you use. Please use GraalVM 23.1.3.
```

Comment:
I remember older versions being harder to find whenever a new version comes out. We should add a direct link in the doc to help people who wish to install it.